charts/aws-fsx-csi-driver/values.yaml

```diff
@@ -7,6 +7,10 @@ image:
   tag: v0.6.0
   pullPolicy: IfNotPresent
 
+# -- Custom labels to add into metadata
+customLabels: {}
```

Comment:
The problem with this implementation is that it does not allow for distinct labels for the node/controller pods, which is important because the Controller and Node services perform different functions.

Reply:
Like that ? we could customize all resources, node and controller : 
```yaml
customLabels:
  global: {}
  node: {}
  controller: {}
```

Reply:
we can always add `controller.customLabels` and `node.customLabels` later. so in sum

`customLabels`
`controller.customLabels` (if needed)
`node.customLabels` (if needed)